packages/tsdl-image/tsdl-image.0.3.0/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "SDL2_Image bindings to go with Tsdl"
+description:
+  "Tsdl_image provides bindings to SDL2_Image intended to be used with Tsdl."
+maintainer: ["Vu Ngoc San <san.vu-ngoc@laposte.net>"]
+authors: ["Julian Squires <julian@cipht.net>"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/sanette/tsdl-image"
+bug-reports: "https://github.com/sanette/tsdl-image/issues"
+depends: [
+  "dune" {>= "1.11"}
+  "ocaml" {>= "4.03"}
+  "tsdl"
+  "ctypes" {>= "0.4.0"}
+  "ctypes-foreign"
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

This unfortunately is a bug of dune < 2.8, so we ned to fix it manually upon submission.
The fix requires a more recent version of dune and updating the dune-projct `(lang dune 2.7)` line. Since this is not a big deal and the CI cheks for it there is no action needed from your side